packages/asc-ui/src/components/LinkList/LinkListStyle.ts

```diff
@@ -1,7 +1,13 @@
 import styled from '@datapunt/asc-core'
+import { color } from '../../utils'
 
-export default styled.ul`
+export interface Props {
```

Comment:
instead, can we use Reac.HTMLAttributes or just `...otherProps`?

Reply:
done